src/Storages/Distributed/DirectoryMonitor.cpp

```diff
@@ -195,6 +200,14 @@ namespace
                             in.getFileName(), distributed_header.revision, DBMS_TCP_PROTOCOL_VERSION);
             }
 
+            if (header_buf.hasPendingData())
+            {
+              readVarUInt(distributed_header.shard_num, header_buf);
+              readStringBinary(distributed_header.cluster, header_buf);
+              readStringBinary(distributed_header.distributed_table, header_buf);
+              readStringBinary(distributed_header.remote_table, header_buf);
```

Comment:
```suggestion
                readVarUInt(distributed_header.shard_num, header_buf);
                readStringBinary(distributed_header.cluster, header_buf);
                readStringBinary(distributed_header.distributed_table, header_buf);
                readStringBinary(distributed_header.remote_table, header_buf);
```

Reply:
Good catch. So the style check in CI can't detect these indent problems?

Reply:
Style check is just a bunch of regexp, no clang-format or something similar.